### ISSUE TITLE
Create tap-friendly checkbox and radio options

### ISSUE THREAD

**Issue by jaredcunha** [uswds/uswds#3767]
Tap-friendly are extremely helpful affordances for making selections on devices, particularly in benefit applications. Advantages include:
- clearly defined hit areas
- greater prominance given to selections made
- visually encapsulates label and form element

**Radio buttons**
From [SBA Certify prototypes](https://ussba.github.io/certify-prototype/2018/09/questionnaire/b3.html)
![Screen Recording 2020-10-15 at 12 35 46 PM](https://user-images.githubusercontent.com/1094951/96161160-ec04c500-0ee4-11eb-888f-a0c7e00029ba.gif)

**Checkbox**
![Screen Recording 2020-10-15 at 12 36 39 PM](https://user-images.githubusercontent.com/1094951/96161238-0343b280-0ee5-11eb-87e1-9ddc94efb17d.gif)

**Inline variant**
From [USDS Multiple Benefit Application Demo](https://usds.github.io/benefits-enrollment-prototype/)
The inline variant might be used when labels are a single word or number perhaps.
![Screen Recording 2020-10-15 at 12 38 32 PM](https://user-images.githubusercontent.com/1094951/96161427-456cf400-0ee5-11eb-8b1f-6134d3b7298d.gif)

**Also found in:**
- [Get CalFresh](https://demo.getcalfresh.org/en)
- [CFPB](https://cfpb.github.io/design-system/components/radio-buttons)

## DESIGN ASSETS
- [x] 10% - Collect and analyze examples, draft research insights, draft list of features
- [x] 20% - Low-fi design review
- [ ] 40% - Hi-fi Sketch assets review
- [ ] 60% - Hi-fi Adobe XD assets review
- [ ] 80% - Sketch and XD assets review w/ Dan + merged into master working files
- [ ] 100% - PR approved by Dan

## GUIDANCE
- [x] 10% - Summarize insights from research (desk + interviews)
- [x] 20% - Draft guidance review 1 (minimum 2 bullets each section)
- [ ] 40% - Get community feedback on draft guidance
- [ ] 60% - Draft guidance review 2
- [ ] 80% - Draft guidance review 3 (w/ Dan)
- [ ] 100% - PR approved by Dan

## CODE
- [x] 10% - Code review 1 (Federalist preview) - interaction design focus
- [x] 20% - Code review 2 (Federalist preview) - visual design focus
- [x] 40% - Get community feedback or conduct tests with users, round 1
- [x] 60% - Code review 3 (Federalist preview) - accessibility focus
- [ ] 80% - Get community feedback or conduct tests with users, round 2
- [ ] 100% - PR approved by Dan

**Comment by jaredcunha** [comment]
The screenshots above are from two separate applications, hence the color differences.

**Comment by debjudy** [comment]
from 10/14 review # 3 on Dan's list

**Comment by jaredcunha** [comment]
Federalist USWDS documentation site links:
Checkbox: https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/accelerator/3767-tap-friendly-targets/components/form-controls/#checkbox

Radio: https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/accelerator/3767-tap-friendly-targets/components/form-controls/#radio-buttons

**Comment by Ben-Judy-Flexion** [comment]
Sketch and XD design assets for checkbox and radio Tile are ready for review and milestone approval by two team members.
I think it would be beneficial for the designers if I walk through how I created these and how I utilized the [Component Design Assets Checklist](https://docs.google.com/document/d/12N_4bxlxsjtjQkvgCqwvWQyECneqE43qsljkmwCEmDE/edit#heading=h.vr55ku6ikg4p) to ensure consistency and quality. Happy to do that whenever we can make time.
![image](https://user-images.githubusercontent.com/56978395/97784522-35077b00-1b6d-11eb-944c-e3033de26adf.png)
![image](https://user-images.githubusercontent.com/56978395/97784524-389b0200-1b6d-11eb-870d-b35292d38286.png)
